homework3/homework3/WebContent/AdminInventary.jsp

```diff
@@ -0,0 +1,40 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+     <%@ taglib uri="WEB-INF/admin-header.tld" prefix= "tag"%>
+    <%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+    <%@ taglib prefix = "fmt" uri = "http://java.sun.com/jsp/jstl/fmt" %>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<link rel="stylesheet" href="foodlover.css">
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<body>
+<tag:Aheader/>
+    <section>
+		<h4>Existing food items</h4>
+		<table border=1 cellpadding=0 cellspacing=0 width=100% >
+            <thead>
+			<tr>
+				<th>Created</th>
+				<th>Item</th>
+				<th>Action</th>	
+			</tr>
+            </thead>
+            <tbody>
+<c:forEach items = "${items}" var="item">
+<tr>
+<td><fmt:formatDate value="${date}" type="time" /></td>
+<td> <img src="${item.getImageUrl()}" width="100" height="100"> <br> ${item.getName()}</td>
+<td><button name="delete">Delete</button> </td>
```

Comment:
this delete button without form or providing the id along with delete button isn't going to work. In other word, the client here doesn't provide enough information for server to know which item user is deleting. 

In specific, you will need to surround this delete button with form and provide the item id via a hidden input tag in order to provide enough information for server to know what it is deleting.